connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConfig.java

```diff
@@ -65,6 +65,8 @@
     public static final String INTERNAL_VALUE_CONVERTER_CLASS_DOC =
             "Converter class for offset value Connect data that implements the <code>Converter</code> interface. Used for converting data like offsets and configs.";
 
+    public static final String CONVERTER_CLASS_CONFIG_DEFAULT = "org.apache.kafka.connect.json.JsonConverter";
```

Comment:
Nit: maybe `JsonConverter.class.getName()` is better? Makes it easier to find usages from an IDE.